packages/common/src/index.js

```
@@ -131,6 +133,11 @@ module.exports = {
   Webhook,
   WebhookDelivery,
   ProRequest,
+  ApiAnalytics,
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)